Rakefile

```diff
@@ -18,5 +18,7 @@ unless Rails.env.production?
   end
 
   task default: %i[spec rubocop scss_lint]
+
+  desc "Lint Ruby and SCSS files"
```

Comment:
I wonder why we don't need a "desc" for the default task too 🤔

Reply:
I was wondering that as well. Looks like [it's a special case](https://github.com/rubocop-hq/rubocop-rake/blob/master/lib/rubocop/cop/rake/desc.rb#L37).